carta/session.py

```diff
@@ -495,6 +495,16 @@ def set_coordinate_system(self, system=CoordinateSystem.AUTO):
         """
         self.call_action("overlayStore.global.setSystem", system)
 
+    def coordinate_system(self):
```

Comment:
shall we make it clear between native coordinate system (as defined in header) and "rendered" coordinate system?

Reply:
This is mostly unambiguous because it's a function on the session, not the image (and will probably soon move to a separate image viewer class). But we can explain this in more detail in the docstring.